# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../rt2500/linux-2.6.17.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur rt2500-1.1.0-b3-orig/Module/rtmp_main.c rt2500-1.1.0-b3/Module/rtmp_main.c
--- rt2500-1.1.0-b3-orig/Module/rtmp_main.c	2005-07-30 16:28:20.000000000 +0000
+++ rt2500-1.1.0-b3/Module/rtmp_main.c	2006-07-01 15:47:11.000000000 +0000
@@ -54,11 +54,11 @@
 //  Global static variable, Debug level flag
 // Don't hide this behind debug define. There should be as little difference between debug and no-debug as possible.
 int debug = 0;	/* Default is off. */
-MODULE_PARM(debug, "i");
+module_param(debug,bool,0);
 MODULE_PARM_DESC(debug, "Enable level: accepted values: 1 to switch debug on, 0 to switch debug off.");
 
 static char *ifname = NULL ;
-MODULE_PARM(ifname, "s");
+module_param(ifname,charp,0);
 MODULE_PARM_DESC(ifname, "Network device name (default ra%d)");
 
 // Following information will be show when you run 'modinfo'
